mlir/utils/performance/perfRunner.py

```diff
@@ -153,21 +197,46 @@ def getMetricArgsForRocprof(arch):
 # Bank conflict functions.The percentage of GPUTime LDS is stalled by bank
 # conflicts. Value range: 0% (optimal) to 100% (bad).
 def getBankConflict(fileName):
+    pass
+
+def getBankConflictV1(fileName):
     if not os.path.exists(fileName):
-        result = "NaN"
-        return result
+        return np.nan
```

Comment:
We had not been consistent and used "nan", "NaN", and np.nan in different places.